vms/mac/README.md

```diff
@@ -0,0 +1,34 @@
+## Creating a new ISO
+1. Download the `Install macOs <VERSION>` app
+   1. Google the OS version that you are trying to make an iso for.
```

Comment:
Use this link? https://support.apple.com/en-us/HT211683

Reply:
I actually find the app in the appstore using google, but I verified the links ex: https://apps.apple.com/us/app/macos-big-sur/id1526878132?mt=12 so will update